packages/react-storage/src/components/StorageManager/utils/uploadFile.ts

```diff
@@ -33,21 +34,20 @@ export function uploadFile({
       contentType,
       onProgress: ({ transferredBytes, totalBytes }) =>
         progressCallback({ transferredBytes, totalBytes }),
+      ...rest,
```

Comment:
We were forgetting some of the other props that can be passed in such as `metadata`, `contentEncoding`

Reply:
Does passing in the additional properties have any impact on the output in regards up the UI?

Reply:
This logic is existing in our current codebase, just got removed on the migration branch, I just didn't notice it when I merged in my migration changes. So it should never have been removed.